modules/observe/analytics/ref-customizing-telemetry-segment.adoc

```diff
@@ -0,0 +1,12 @@
+[id="customizing-telemetry-segment_{context}"]
+= Customizing Segment source
+
+
+The `analytics-provider-segment` plugin sends the collected web analytics data to {company-name} by default. However, you can configure a new Segment source that receives web analytics data based on your needs. For configuration, you need a unique Segment write key that points to the Segment source.
+
+[NOTE]
+====
+By configuring a new Segment source, you can collect and analyze the same set of data that is mentioned in link:{telemetry-data-collection-book-url}[{telemetry-data-collection-book-title}]. You might also require to create your own web analytics data collection notice for your application users.
```

Comment:
Modularity: 

* A link denotes an **Additional resources*** section. Consider moving the first sentence to a list item in the **Additional resources** section.
* "require" in a sentence denotes a **Prerequisites** section. Consider moving the second sentence to a list item in the **Prerequisites** section.